### PR TITLE
Add php

### PR DESCRIPTION
**Pull request by sikesLpp**
Configuration file for govharvester
Fixed php tags

### CODE REVIEW

File: src/php/gov-harvester/govharvester.php [hunk 1]
```diff
@@ -169,10 +264,25 @@ function docParsePage($domdoc)
...
+	// construct a sensible filename
+	$filenumber = substr(strrchr(dirname($path),'/'),1);
+
+	// download the file
+	file_put_contents($confarray['destinationdir']. '/' . $filenumber . '_' . basename($path), fopen("https://www.gov.uk" . $path, 'r'));
+}
+
 
-?>
+php?>
```
Comment (eddiejaoude) [inline src/php/gov-harvester/govharvester.php]: > The closing ?> tag MUST be omitted from files containing only PHP.

Closing `php` tag not required

Reference: http://www.php-fig.org/psr/psr-2/#2-2-files

Comment (sikesLpp) [reply src/php/gov-harvester/govharvester.php]: done

File: src/php/gov-harvester/govharvester.php [hunk 2]
```diff
@@ -8,39 +8,131 @@
 // * GNU public license v3
 // *************************************************************************************************
 
+// disable irritant useless messages
+error_reporting(E_ALL ^ E_NOTICE ^ E_WARNING );
```
Comment (eddiejaoude) [inline src/php/gov-harvester/govharvester.php:12]: This is not recommended practice.

File: src/php/gov-harvester/govharvester.php [hunk 3]
```diff
@@ -8,39 +8,131 @@
...
+if($args['f'] != '')
+	$confilepath = $args['f'];
+else
+	$confilepath = 'govharvester.conf';
+
+// attempt to parse config file
+if(!$confarray = confParseFile($confilepath))
 {
-	echo "Could not open or parse document\n";
 	exit;
```
Comment (eddiejaoude) [inline src/php/gov-harvester/govharvester.php:26]: `exit` alone will not provide feedback on what went wrong

File: src/php/gov-harvester/govharvester.php [hunk 4]
```diff
@@ -8,39 +8,131 @@
...
 }
 
+
+print_r($confarray);
+//exit;
```
Comment (eddiejaoude) [inline src/php/gov-harvester/govharvester.php:31]: Debug code left in & commented out?

**Comment by elischutze** [comment]
have any of the comments been addressed here? can we merge this @sikesLpp @eddiejaoude

**Pull request merged**